_drafts/2023-02-13-cheating-the-system-for-fun-and-profit.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+permalink: cheating-the-system-for-fun-and-profit
+title: 'Cheating the System for Fun and Profit'
+tags: [accessibility]
+---
+
+Okay, that title might just be a little bit of clickbait. The idea is to use
```

Comment:
I think I'd expand this intro a little to explain why we might want to do this. ie, it gives us more information from VO in the simulator than we get from the Ax inspector.